helpers/lib/cdnify.js

```diff
@@ -60,16 +60,20 @@ module.exports = globals => {
             return path;
         }
 
-        if (path[0] !== '/') {
-            path = '/' + path;
-        }
 
         if (!versionId) {
+            if (path[0] !== '/') {
```

Comment:
Instead of nesting can we just return path early if version exists ?

Reply:
This logic is already convoluted enough, I think this is the best we can do right now without a much bigger refactor.